src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div className="App">
+      <Display />
+      <ButtonPanel />
+    </div>
```

Comment:
The children components inside the `App` component should be wrapped in <a href="https://reactjs.org/docs/fragments.html#short-syntax" rel="nofollow noreferrer noopener" target="_blank">React.Fragment</a> short syntax (<a href="https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png">screenshot</a> from <a href="https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/" rel="nofollow noreferrer noopener" target="_blank">Pathwright</a>)